packages/components/src/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import { Story, ComponentMeta } from '@storybook/react'
+import { Accordion, IAccordionProps } from './Accordion'
+import React from 'react'
+
+export default {
+  title: 'Data/Accordion',
+  component: Accordion,
+  parameters: {
+    docs: {
+      description: {
+        component: `\`<Accordion />\` `
```

Comment:
Descripting the component would be nice. If you need inspiration, you can see what kind of text [Gov.uk](https://design-system.service.gov.uk/components/accordion/) or [MUI](https://mui.com/material-ui/react-accordion/) use here.